newscout_web/dashboard/templates/dashboard-article-edit.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'dashboard-base.html' %}
```

Comment:
Make sure to create `dashboard` sub-dir within `templates` to have all grouped templates into one directory

Reply:
it is not required as templates directory is already in dashboard app. i.e. dashboard/templates/